### DISCUSSION TITLE
Does Rich work with print f?

### DISCUSSION THREAD

**Discussion by Cleman25** [Textualize/rich#463] [Q&A]
![image](https://user-images.githubusercontent.com/5369686/100955605-0d8d3080-34d4-11eb-955f-499e064786bd.png)
Will rich work with something like that? I've tested it and it doesn't seem to work in PyCharm's terminal. Not sure if I'm doing something wrong

**Comment by willmcgugan** [comment]
Yes, assuming you've imported rich.print there. But see the note about PyCharm in the [introduction](https://rich.readthedocs.io/en/latest/introduction.html)

**Reply by Cleman25** [reply]
Sorry for the late response. I ended up getting it to work. Seems the issue was PyCharm's "Run" tab in the refused to display it. I ran it on the terminal and it worked.
